README.md

```diff
@@ -1 +1,101 @@
-# shin_capstone
\ No newline at end of file
+# shin_capstone
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Capstone Project
+
+> This is a module 1: week 5 project in Microverse full stack course.
+
+
```

Comment:
- [ ] Please describe very well what your website is about here in the project description section. From what I have seen, I would suggest you say like 'The Clickbank website is a platform that hosts affiliate marketing conferences and events. You can browse the featured speakers...' so, please describe the project very well in this section 😃 👍